data/korath/korath ships.txt

```diff
@@ -1927,7 +1927,7 @@ ship "Far Osk 27"
 	gun 13 -8
 	explode "tiny explosion" 25
 	explode "small explosion" 15
-	description "Because it is piloted by a computer and has no need for a cockpit or life support systems, the FS27 fighter is able to carry far more weaponry than any comparable human ship."
+	description "Because it is piloted by a computer and has no need for a cockpit or life support systems, the FS27 fighter is able to support more weaponry than any comparable human ship."
```

Comment:
What about this?

```suggestion
	description "Because it is piloted by a computer and has no need for a cockpit or life support systems, the FS27 fighter is able to support a thicker hull, more shields, more powerful guns than any comparable human ship."
```

Reply:
~~missing a conjunction~~

Reply:
That's what happens when you post things when you should be in bed sleeping.  LOL